draw/mouse.go

```diff
@@ -50,7 +50,7 @@ func mouseproc(mc *Mousectl, d *Display, ch chan Mouse, rch chan bool) {
 		if resized {
 			rch <- true
 		}
-		mm := Mouse{image.Point{m.X, m.Y}, m.Buttons, uint32(m.Msec)}
+		mm := Mouse{image.Point{int(int32(m.X)), int(int32(m.Y))}, m.Buttons, uint32(m.Msec)}
```

Comment:
The right fix is to do the cast in `bit.go`, when we're decoding the coordinates from `devdraw`:
```
func gbit32(b []byte) (int, []byte) {
	return int(int32(uint32(b[3]) | uint32(b[2])<<8 | uint32(b[1])<<16 | uint32(b[0])<<24)), b[4:]
}
```